experiments/run_slicegpt_perplexity.py

```diff
@@ -233,15 +234,24 @@ def reset_model_device() -> None:
     rotate.rotate_and_slice(model_adapter, train_loader, scheduler, final_orientation=args.final_orientation)
 
     if args.save_dir:
-        path = pathlib.Path(args.save_dir)
-        path.mkdir(parents=True, exist_ok=True)
+        sliced_model_dir = pathlib.Path(args.save_dir)
+        sliced_model_dir.mkdir(parents=True, exist_ok=True)
 
-        model_path = path / f'{pathlib.Path(args.model).name}_{args.sparsity}.pt'
+        sliced_model_name = sliced_model_dir / f'{pathlib.Path(args.model).name}_{args.sparsity}.pt'
 
-        config_path = model_path.with_suffix('.json')
+        config_path = sliced_model_name.with_suffix('.json')
         config_path.write_text(model_adapter.slicing_conf.to_json_string())
 
-        torch.save(model.state_dict(), model_path)
+        # If slicing a local model, also save HF config files in sliced model dir
+        if args.model_path:
+            # copy all config files
+            for file in glob.glob(f"{args.model_path}/*config*.json"):
```

Comment:
Could use Path's .glob() method I believe and avoid extra import.

Reply:
Done (I think I forgot to push earlier)